api/watch/plan.go

```diff
@@ -118,13 +119,21 @@ OUTER:
 		p.lastResult = result
 		// If a hybrid handler exists use that
 		if p.HybridHandler != nil {
+			if first == true && p.FireOnCreate == "no" {
```

Comment:
Trying to be explicit with `first == true`. Please let me know if you want it changed. :)